### PR TITLE
Add CVE-2015-0250

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by vasulakkaraju** [comment]
This is CVE is duplication for the CVE-2017-5662

File: database/java/2015/0250.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - http://www.ubuntu.com/usn/USN-2548-1
+    - http://xmlgraphics.apache.org/security.html
+
+affected:
+    - groupId: org.apache.xmlgraphics
+      artifactId: batik-svg-dom
+      version:
+        - "<= 1.7"
+      fixedin:
+        - "== 1.8"
```
Comment (msrb) [inline database/java/2015/0250.yaml]: should be >= (all newer versions will carry the fix as well), and no whitespace.

**Comment by vasulakkaraju** [comment]
@msrb updated as suggested.Thanks

**Comment by msrb** [comment]
LGTM, thanks :wink:

**Pull request merged**